pypeit/core/flexure.py

```diff
@@ -388,6 +391,7 @@ def spec_flexure_obj(specobjs, maskslits, method, sky_file, mxshft=None):
                 msgs.info("No flexure corrections could be made")
                 break
             # Setup
+            embed(header='394 of flexure')  # JXP -- Make sure this is correct!!
```

Comment:
Should this be here?

Reply:
have changed to an `error`.  Am pretty sure 
this isn't code quite right anymore but it appears
we have nothing testing it at the moment..